bridge/elrond/client.go

```diff
@@ -47,80 +47,104 @@ func (e QueryResponseErr) Error() string {
 	return fmt.Sprintf("Got response code %q and message %q", e.code, e.message)
 }
 
-type elrondProxy interface {
-	GetNetworkConfig() (*data.NetworkConfig, error)
-	SendTransaction(*data.Transaction) (string, error)
-	GetTransactionInfoWithResults(hash string) (*data.TransactionInfo, error)
-	ExecuteVMQuery(vmRequest *data.VmValueRequest) (*data.VmValuesResponseData, error)
+// Client represents the Elrond Client implementation
+type Client struct {
```

Comment:
why almost everything in this file is exported?

Reply:
will be refactored

Reply:
done, part of it, added TODO